### PR TITLE
Add smart_proxy_ansible [deb]

### PR DESCRIPTION
**Pull request by iNecas**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.13
* [ ] 1.12
* [ ] 1.11

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

### CODE REVIEW

File: plugins/smart_proxy_ansible/control [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Source: ruby-smart-proxy-ansible
+Section: ruby
+Priority: optional
+Maintainer: Ivan Ne�as <inecas@redhat.com>
```
Comment (mmoll) [inline plugins/smart_proxy_ansible/control]: it looks likes this was not written in UTF-8?

File: plugins/smart_proxy_ansible/changelog [hunk 2]
```diff
@@ -0,0 +1,5 @@
+ruby-smart-ansible (1.0.0-1) unstable; urgency=low
```
Comment (mmoll) [inline plugins/smart_proxy_ansible/changelog]: +proxy

File: plugins/smart_proxy_ansible/control (hunk 1 above)
Comment (iNecas) [inline plugins/smart_proxy_ansible/control]: updated

File: plugins/smart_proxy_ansible/changelog (hunk 2 above)
Comment (iNecas) [inline plugins/smart_proxy_ansible/changelog]: updated

**Comment by mmoll** [comment]
cherry-picked as 640dbd0 and b1a0fcd, díky @iNecas!

**Pull request closed without merging**